packages/sfui/frameworks/react/components/SfTextarea/SfTextarea.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import { SfTextareaSize, useFocusVisible } from '@storefront-ui/react';
+import { forwardRef } from 'react';
+import type { SfTextareaProps } from './types';
+
+const sizeClasses = {
+  [SfTextareaSize.sm]: ' h-[32px] py-[6px] pl-4 pr-3',
+  [SfTextareaSize.base]: 'h-[64px] py-2 pl-4 pr-3',
+  [SfTextareaSize.lg]: 'h-[72px], p-3 pl-4',
+};
+
+export default forwardRef<HTMLTextAreaElement, SfTextareaProps>(
+  ({ size = SfTextareaSize.base, invalid = false, className, ...attributes }, ref): JSX.Element => {
+    const { isFocusVisible } = useFocusVisible({ isTextInput: true });
+
+    return (
+      <textarea
+        ref={ref}
+        className={classNames([
+          'flex items-center px-4 bg-white rounded-md text-neutral-900 hover:ring-primary-800 focus-within:caret-primary-700 active:caret-primary-700 active:ring-primary-700 active:ring-2 focus-within:ring-primary-700 focus-within:ring-2',
```

Comment:
```suggestion
          'block items-center px-4 bg-white rounded-md text-neutral-900 hover:ring-primary-800 focus-within:caret-primary-700 active:caret-primary-700 active:ring-primary-700 active:ring-2 focus-within:ring-primary-700 focus-within:ring-2',
```
like I wrote in vue component block could be used instead of flex

Reply:
removed